tests/test_surface_events.cpp

```diff
@@ -495,6 +495,52 @@ TEST_F(ClientSurfaceEventsTest, surface_moves_while_under_pointer)
         });
 }
 
+TEST_F(ClientSurfaceEventsTest, frame_timestamp_increases)
+{
+    using namespace testing;
+
+    Test::RecordProperty("wlcs-skip-test", "Mir does not send enter events yet");
```

Comment:
We should not be implementing Mir specific limitations in these tests.

Any tests that Mir expects to fail ought to be "xfail"ed by the Mir integration code.

Reply:
fixed